jenkins-build-collector/src/test/java/com/capitalone/dashboard/collector/HudsonCollectorTaskTests.java

```diff
@@ -116,7 +116,8 @@ public void delete_job() {
     @Test
     public void delete_never_job() {
         HudsonCollector collector = collectorWithOneServer();
-        collector.setId(new ObjectId().get());
+        new ObjectId();
```

Comment:
Good catch, I fixed this, it is a strange fix done by eclipse, just deleted the new ObjectID(); it is not required.